### PR TITLE
Irminger 10 Gliders

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

**Comment by cdobs** [comment]
Sensor configurations can be reviewed in FCPS:

898 - 3304-00160-00064
486 - 3304-00160-00065
575 - 3304-00170-00040

File: deployment/GI05MOAS-G0898_Deploy.csv [hunk 1]
```diff
@@ -0,0 +1,5 @@
+CUID_Deploy,deployedBy,CUID_Recover,recoveredBy,Reference Designator,deploymentNumber,versionNumber,startDateTime,stopDateTime,mooring.uid,node.uid,sensor.uid,lat,lon,orbit,deployment_depth,water_depth,notes,electrical.uid,assembly_template_revision
+AR76-03,,,,GI05MOAS-G0898-00-ENG000000,1,1,2023-08-30T15:00:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00898,CGCON-GLDROO-00898,59.8827,-39.0366,,1000,1000,,,
+AR76-03,,,,GI05MOAS-G0898-01-FLORDM000,1,1,2023-08-30T15:00:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00898,CGINS-FLORDM-06989,59.8827,-39.0366,,1000,1000,,,
+AR76-03,,,,GI05MOAS-G0898-02-DOSTAM000,1,1,2023-08-30T15:00:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00898,CGINS-DOSTAM-00685,59.8827,-39.0366,,1000,1000,top mount,,
+AR76-03,,,,GI05MOAS-G0898-04-CTDGVM000,1,1,2023-08-30T15:00:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00898,CGINS-CTDGVM-09678,59.8827,-39.0366,,1000,1000,,,
```
Comment (rtravis-whoi) [inline deployment/GI05MOAS-G0898_Deploy.csv:5]: looks good

File: deployment/GI05MOAS-GL486_Deploy.csv [hunk 2]
```diff
@@ -2,4 +2,8 @@ CUID_Deploy,deployedBy,CUID_Recover,recoveredBy,Reference Designator,deploymentN
...
-AR21,,AR21,,GI05MOAS-GL486-04-CTDGVM000,2,1,2017-08-09T09:55:00,2017-08-09T14:19:00,CGVGI-05MOAS-00000,CGVEH-GLDROO-00486,CGINS-CTDGVM-09174,59.8617,-39.1491,,1000,1000,glider leaked - was recovered on the same day,,
\ No newline at end of file
+AR21,,AR21,,GI05MOAS-GL486-04-CTDGVM000,2,1,2017-08-09T09:55:00,2017-08-09T14:19:00,CGVGI-05MOAS-00000,CGVEH-GLDROO-00486,CGINS-CTDGVM-09174,59.8617,-39.1491,,1000,1000,glider leaked - was recovered on the same day,,
+AR76-03,,,,GI05MOAS-GL486-00-ENG000000,3,1,2023-08-30T15:04:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00486,CGCON-GLDROO-00486,59.8827,-39.0369,,1000,1000,,,
+AR76-03,,,,GI05MOAS-GL486-01-FLORDM000,3,1,2023-08-30T15:04:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00486,CGINS-FLORDM-03551,59.8827,-39.0369,,1000,1000,,,
+AR76-03,,,,GI05MOAS-GL486-02-DOSTAM000,3,1,2023-08-30T15:04:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00486,CGINS-DOSTAM-00566,59.8827,-39.0369,,1000,1000,top mount,,
+AR76-03,,,,GI05MOAS-GL486-04-CTDGVM000,3,1,2023-08-30T15:04:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00486,CGINS-CTDGVM-09174,59.8827,-39.0369,,1000,1000,,,
```
Comment (rtravis-whoi) [inline deployment/GI05MOAS-GL486_Deploy.csv:9]: looks good

File: deployment/GI05MOAS-PG575_Deploy.csv [hunk 3]
```diff
@@ -5,4 +5,11 @@ AR69-01,,AR69-01,,GI05MOAS-PG575-02-DOSTAM000,3,1,2022-06-28T17:24:00,2022-07-01
...
+AR76-03,,,,GI05MOAS-PG575-00-ENG000000,4,1,2023-08-30T15:10:00,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00575,CGCON-GLDRGP-00575,59.8827,-39.0369,,200,200,,,
+AR76-03,,,,GI05MOAS-PG575-01-CTDGVM000,4,1,2023-08-30T15:10:00,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00575,CGINS-CTDGVM-09287,59.8827,-39.0369,,200,200,,,
+AR76-03,,,,GI05MOAS-PG575-02-DOSTAM000,4,1,2023-08-30T15:10:00,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00575,CGINS-DOSTAM-00414,59.8827,-39.0369,,200,200,top mount,,
+AR76-03,,,,GI05MOAS-PG575-03-FLORTM000,4,1,2023-08-30T15:10:00,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00575,CGINS-FLORTM-03815,59.8827,-39.0369,,200,200,,,
+AR76-03,,,,GI05MOAS-PG575-04-FLORTO000,4,1,2023-08-30T15:10:00,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00575,CGINS-FLORTO-01332,59.8827,-39.0369,,200,200,,,
+AR76-03,,,,GI05MOAS-PG575-05-NUTNRM000,4,1,2023-08-30T15:10:00,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00575,CGINS-NUTNRM-00727,59.8827,-39.0369,,200,200,,,
+AR76-03,,,,GI05MOAS-PG575-06-PARADM000,4,1,2023-08-30T15:10:00,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00575,CGINS-PARADM-50200,59.8827,-39.0369,,200,200,,,
```
Comment (rtravis-whoi) [inline deployment/GI05MOAS-PG575_Deploy.csv:15]: looks good

**Review by rtravis-whoi — APPROVED** [review]

**Review by reedan88 — APPROVED** [review]
Looks good.

**Comment by rtravis-whoi** [comment]
[Irminger 10 AM Updates](https://trello.com/c/n3g5jRk2/198-irminger-10-am-updates)

**Pull request merged**